reset_password/reset_password.php

```diff
@@ -13,19 +13,30 @@
     case "reset_password_request":
       $email = get_user_by_mail($conn, $reqObj->email);
       if (!$email) {
-        echo json_encode(["error" => true, "errorText" => "E-Mail not found"]);
+        echo json_encode(["error" => true, "errorText" => "E-Mail not found."]);
+        exit;
       } else {
         $id = $email["ID"];
         $email = $email["EMail"];
 
+        #only one email every 10 minutes
```

Comment:
our comments in general are useless. Remove alle comments or comment every logic